api/src/main/java/net/md_5/bungee/api/event/ChatEvent.java

```diff
@@ -39,4 +41,16 @@ public boolean isCommand()
     {
         return message.length() > 0 && message.charAt( 0 ) == '/';
     }
+
+    /**
+     * Checks whether this message is run on this proxy server
+     *
+     * @return if this command runs on the proxy
+     */
+    public boolean isProxyCommand()
+    {
+        String commandName = message.split(" ",2)[0].substring( 1 );
```

Comment:
indexOf(" ") -> substring

Reply:
Great idea! I've never thought of using that faster method!